crates/starknet-devnet-core/src/starknet/predeployed.rs

```diff
@@ -51,10 +50,15 @@ pub(crate) fn initialize_erc20_at_address(
         // necessary to set - otherwise minting txs cannot be executed
         ("Ownable_owner", Felt::from_prefixed_hex_str(CHARGEABLE_ACCOUNT_ADDRESS)?),
     ] {
-        let storage_var_address = get_storage_var_address(storage_var_name, &[])?.try_into()?;
+        let storage_var_address = starknet_types::patricia_key::PatriciaKey::new(
+            Felt::new(get_storage_var_address(storage_var_name, &[]).unwrap().to_bytes_be())
+                .unwrap(),
+        )
+        .unwrap();
```

Comment:
Let's not have such `unwrap`s in production code.

Reply:
Done